lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -52,6 +61,26 @@ static inline void skip_command_prefix(const char **cmd)
 	(*cmd)++;
 }
 
+static inline bool check_response_for_forced_string(const char *tmpstr)
+{
+	bool retval = false;
+
+	if (!strncmp(tmpstr, "%HWVERSION", AT_CMD_HWVERSION_LEN)) {
+			retval = true;
+	}
+	else if (!strncmp(tmpstr, "%SHORTSWVER", AT_CMD_SHORTSWVER_LEN)) {
```

Comment:
This should be at the same line as closing bracket, i. e. `} else if (...) {`.

Or personally I'd even consider making this whole expression shorter:
```
if (!strncmp(tmpstr, "%HWVERSION", AT_CMD_HWVERSION_LEN) ||
    !strncmp(tmpstr, "%SHORTSWVER", AT_CMD_SHORTSWVER_LEN) ||
    !strncmp(tmpstr, "%XMODEMUUID", AT_CMD_XMODEMUUID_LEN) ||
    !strncmp(tmpstr, "%XICCID", AT_CMD_XICCID_LEN)) {
	retval = true;
}
```

Reply:
Agreed. Fixed.